Common/include/config_structure.hpp

```diff
@@ -417,6 +417,7 @@ class CConfig {
   unsigned short nGridMovement;		/*!< \brief Number of grid movement types specified. */
   unsigned short nTurboMachineryKind; 	/*!< \brief Number turbomachinery types specified. */
   unsigned short nParamDV;		/*!< \brief Number of parameters of the design variable. */
+  string DV_Filename;      /*!< \brief Filename for providing surface positions from an external parameterization. */
```

Comment:
I used a similar approach for structural DV (e.g., stiffness properties in different regions of the structural domain). Maybe we should seek to make it homogeneous (although that would bring some discussion about the python framework for optimization).

Reply:
Agreed. This topic (generalizing the python framework to arbitrary DVs), standardized BC data input for a flexible number of variables, and the output are all at the top of my list for what we need to discuss as a development team soon.

Reply:
Great! I see we are in the same page.
I'm doing some preliminary work on the output, I'll share it when I have some working implementation.